src/components/movie-details.tsx

```diff
@@ -0,0 +1,76 @@
+                        <span><span className={styles['movie-details-company-name-bold']}>netflix</span>roulette</span>
+                        <Link
+                            href={{ pathname: `/`, query: props.searchParams }} style={{ textDecoration: 'none' }}
+                        >
+                            <button
+                                className={styles['movie-details-search-movie-button']}
+                            // onClick={() => router.push('/')}
+                            >
+                                X
+                            </button>
+                        </Link>
+                    </div >
+                    <div className="row">
+                        <div className="col span4">
+                            <img className={styles['movie-details-image']} src={movie.imageUrl} />
+                        </div>
+                        <div className="col space"></div>
+                        <div className="col span5">
+                            <div className="row">
+                                <div className={`col span0 ${styles['movie-details-name']}`}>
+                                    <span id='movie-details-name'>{movie.name.toUpperCase()}</span>
+                                </div>
+                                <div className="col space"></div>
+                                <div className={`col ${styles['movie-details-number-circle']}`}>
+                                    {movie.rating}
+                                </div>
+                            </div>
+                            <div className="row">
+                                <div className="col span0 ${styles['movie-details-genre']}">
+                                    <span>{movie.relevantGenres.map((genre: string) => capitalize(genre)).join(', ')}</span>
+                                </div>
+                            </div>
+                            <div className="row">
+                                <div className={`col span0 ${styles['movie-details-release']}`}>
+                                    <span>{movie.releaseYear}</span>
+                                </div>
+                                <div className="col space"></div>
+                                <div className={`col span0 ${styles['movie-details-release']}`}>
+                                    <span>{movie.duration}</span>
+                                </div>
+                            </div>
+                            <div className="row">
+                                <div className={`col span10 ${styles['movie-details-description']}`}>
+                                    <span>{movie.description}</span>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div >
+                : <h1>Incorrect movie ID: {props.movieId}</h1>
```

Comment:
for such a big block it seems to be better `if (!movie) return <h1>Incorrect movie ID: {props.movieId}</h1>` in line 18